backend/experiment/rules/gold_msi.py

```diff
@@ -27,7 +26,14 @@ def __init__(self):
         self.questions = MSI_F3_MUSICAL_TRAINING + demographics
 
     def first_round(self, experiment):
-        consent = Consent()
+        consent = Consent(
+            style=FrontendStyle(
+                FrontendStyle.BOOLEAN,
+                submit_button=FrontendStyle(
```

Comment:
What is the reasoning here to assign the `SUCCESS` style here for the submit button? I'd assume it's already implicated in the `BOOLEAN` style at a higher level?

Reply:
That's just in case you would want to override it. The two buttons in the `Consent` form aren't styled using the button array component afaik.

Reply:
You're right, so I think we should refactor such that Consent reuses the button array component, and we can also set styles in the same way.

Reply:
I think that would be a good follow up ticket as it seems more like a new feature/refactor and I would like to keep that separate. However, I will remove the nested styles from the consent then in `gold_msi.py`.